src/materialized/tests/pgwire.rs

```diff
@@ -246,13 +247,16 @@ fn test_persistence() -> Result<(), Box<dyn Error>> {
                 .query("SHOW INDEXES FROM mat", &[])?
                 .into_iter()
                 .map(|row| (row.get("Column_name"), row.get("Seq_in_index")))
-                .collect::<Vec<(String, i64)>>(),
-            &[
+                .collect::<HashSet<(String, i64)>>(),
```

Comment:
Should we guarantee an ordering here? I think we do for the other `SHOW` commands. (Maybe you mentioned this in the PR description?) Since you can't specify an `ORDER BY` on a `SHOW` it seemed useful to just pick something sensible to order by (the name, I think?) and force it.

Reply:
I took a very lazy but I think correct/sufficient approach here of ordering on the entire column set from left-to-right, lmk if you think something more sophisticated is warranted (this seems to give a sensible ordering, at least)

Reply:
No I think this is exactly right, thanks!